src/it/scala/org/ergoplatform/it/container/Docker.scala

```diff
@@ -71,9 +71,9 @@ class Docker(suiteConfig: Config = ConfigFactory.empty,
   def startNodes(nodeConfigs: List[Config],
                  configEnrich: ExtraConfig = noExtraConfig): Try[List[Node]] = {
     log.trace(s"Starting ${nodeConfigs.size} containers")
-    val nodes: Try[List[Node]] = nodeConfigs.map(cfg => startNode(cfg, configEnrich)).sequence
-    blocking(Thread.sleep(nodeConfigs.size * 5000))
```

Comment:
@mike-aksarin Is there any reason to prefer hardcoded duration here? Rather than trying to tune the hardcoded time to wait for containers to start I've used `waitForStartupBlocking` (our interpreter is quite heavy ATM, instantiation time varies a lot).